apps/opik-backend/src/test/java/com/comet/opik/api/resources/utils/resources/PromptResourceClient.java

```diff
@@ -0,0 +1,65 @@
+package com.comet.opik.api.resources.utils.resources;
+
+import com.comet.opik.api.CreatePromptVersion;
+import com.comet.opik.api.Prompt;
+import com.comet.opik.api.PromptVersion;
+import com.comet.opik.api.resources.utils.TestUtils;
+import com.comet.opik.infrastructure.auth.RequestContext;
+import jakarta.ws.rs.client.Entity;
+import jakarta.ws.rs.core.HttpHeaders;
+import org.apache.hc.core5.http.HttpStatus;
+import ru.vyarus.dropwizard.guice.test.ClientSupport;
+import uk.co.jemos.podam.api.PodamFactory;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public interface PromptResourceClient {
```

Comment:
We could use it for other items as well, not only Prompts, there are a lot of duplication. Not in this PR, but as a refactoring later.

Reply:
This kind of class yes, but I would keep a separate class per resource to separate concerns and keep it simple